pattern_importer.py

```
@@ -0,0 +1,322 @@
+                if rule_level >= min_level:
+                    pattern_id = rule.get('id', '')
+                    if not pattern_id:
+                        pattern_id = rule.get('name', '').lower().replace(' ', '_').replace('-', '_')
+                    
+                    regex_pattern = rule.get('pattern') or rule.get('regex')
+                    
+                    if regex_pattern and pattern_id:
+                        patterns[pattern_id] = regex_pattern
+            
+            logger.info(f"Loaded {len(patterns)} patterns from secrets-patterns-db (min confidence: {min_confidence})")
+            return patterns
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)